src/wwwroot/partials/settings/my-plan.html

```diff
@@ -7,21 +7,21 @@
   </div>
   <div class="section--wrapper my-plan--info-container">
     <div class="item flex flex--wrap first">
-      <div class="flex one flex--wrap">
+      <div class="flex one flex--wrap current-plan-price">
```

Comment:
Is  an interesting, for the moment I proceed to change the name of class by `price` and in the future, we will investigate as to create a locator in page object by `Data Attribute` because doesn't have an example in the WebApp